.gitattributes

```diff
@@ -1,3 +1 @@
 yarn.lock linguist-generated
-.yarn/releases/** filter=lfs diff=lfs merge=lfs -text linguist-generated
-.yarn/plugins/** filter=lfs diff=lfs merge=lfs -text linguist-generated
```

Comment:
Yarn 4 comes with a lot more plugins bundled and uses Node's corepack for version/release management.